README.md

```diff
@@ -44,6 +44,16 @@ To support multiple devices, or permit overriding default behavior, you may pref
 `${PROBE_RUN_CHIP}` environment variable, and set `runner` (or
 `CARGO_TARGET_${TARGET_ARCH}_RUNNER`) to `probe-run`.
 
+If you have several probes connected, you can specify which one to use adding
```

Comment:
could you add a note here indicating the syntax for the `--probe` flag? or say that the syntax can be found in the output of the `-h` command

Reply:
OK, done.